gear/csv_center_splitter/src/docker/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "csv-center-splitter",
+    "label": "CSV Center Splitter",
+    "description": "Splits a CSV by center ID",
+    "version": "0.1.0",
+    "author": "NACC",
+    "maintainer": "NACC <nacchelp@uw.edu>",
+    "cite": "",
+    "license": "MPL-2.0",
+    "url": "https://naccdata.github.io/flywheel-gear-extensions",
+    "source": "https://github.com/naccdata/flywheel-gear-extensions",
+    "environment": {
+        "FLYWHEEL": "/flywheel/v0"
+    },
+    "custom": {
+        "gear-builder": {
+            "category": "utility",
+            "image": "naccdata/csv-center-splitter:0.1.0"
+        },
+        "flywheel": {
+            "suite": "Curation",
+            "show-job": true
+        }
+    },
+    "inputs": {
+        "api-key": {
+            "base": "api-key"
+        },
+        "input_file": {
+            "description": "The input CSV",
+            "base": "file",
+            "type": {
+                "enum": [
+                    "tabular data"
+                ]
+            }
+        }
+    },
+    "config": {
+        "dry_run": {
+            "description": "Whether to do a dry run",
+            "type": "boolean",
+            "default": false
+        },
+        "adcid_key": {
+            "description": "Column name from the input CSV with the ADCIDs",
+            "type": "string"
+        },
+        "target_project": {
```

Comment:
Project name, since it can be for multiple centers. I can update the description to clarify. `staging_project_id` is the one where it's an ID and staged at a single project and is intended to override `target_project` if specified.